.github/workflows/build.yml

```diff
@@ -270,6 +270,9 @@ jobs:
     needs: [  build  ]
     if: github.ref != 'refs/heads/master'
     runs-on: ubuntu-latest
+    concurrency: Review_${{github.event.inputs.pr}}
```

Comment:
what happens if not PR? ( github.event.inputs.pr)  e.g main\dev branch -, why not use `${{ github.ref }}`?

Reply:
If it were not a PR it would not be in the review stage, the workflow only triggers on  workflow_dispatch  pull_request push:    branches: [ master ] ( workflow dispatch is only used for debugging ) and the review job only triggers on NOT master branches.  so it will only hit this job if a PR